### PR TITLE
Rebalances the deep space storage bunker space ruin.

### PR DESCRIPTION
**Pull request by WJohn**
Because 6 boxes of 9mm ammo (180 rounds!) and two wt-550s is just dumb, especially if you got the 9mm APS pistol from the caravan ruin.

Reduced that to 3 magazines (45 rounds) and one wt-550 with one spare compatible mag for it (so you cannot dual wield that, and also have to spend time reloading instead of just pulling out another gun).

~~Also reduced the amount of food in the food storage by half and made it look a bit neater (I do love me some stacked looking items in crates and racks).~~ Undid this at ninjanomnom's request. Still kept neatly stacked, though.

Lastly, turned the syndicate hardsuit into a regular grey space suit. We don't need non-antags freaking people out over it and it's very, very strong space armor to boot. It was also weirdly out of place with the aesthetic of the ruin, being the *only* thing that pointed to it being syndicate base at all.

:cl: WJohnston
balance: Reduced the 180 rounds (6 boxes) of 9mm ammo in the deep storage bunker to 45 (3 mags). Removed one of the wt-550s but put a compatible magazine in its place for the other to use. Turned the syndicate hardsuit into a regular grey space suit.
/:cl:

### CODE REVIEW

**Comment by Hatterhat** [comment]
wack

**Comment by subject217** [comment]
>We don't need non-antags freaking people out over it and it's very, very strong space armor to boot.
Miner legion loot does it anyway. At any rate I always felt the hardsuit was pretty harmless, doesn't really need to go IMO.

**Comment by duckay** [comment]
it could do with turrets, but removal of the only guaranteed method to get a bloodred is hurty and painy
wack

**Comment by WJohn** [comment]
Miner's equipment and loot is not a good baseline to compare anything in the game to. It is in its own league.

**Comment by Tlaltecuhtli** [comment]
9mm ammo is autolathe printable, the sec auto rifles (they use 4.57 ammo not 9mm ammo) are garbage as they are pretty much ballistic generic lasers, nuke op suit which is somewhat worth wasting half the round navigating space and hacking 3 strong doors is a free ticket to being stripped by sec and looted also anti meta

**Comment by Grazyn** [comment]
Deep storage has a GPS (DEEP tag IIRC) so it's not really that hard to find

**Comment by Tlaltecuhtli** [comment]
you still have to go trough randomized maps to get in the right place, get jetpack gloves and other stuff to start exploring

**Comment by AlexTheSysop42** [comment]
> only thing that pointed to it being syndie

Pretty sure the notes on the bulletin board do too

**Comment by Anonmare** [comment]
Remove the GPS and add turrets

**Comment by WJohn** [comment]
@Grazyn Nope, none of the notices point to that at all.

**Comment by Supermichael777** [comment]
>everyone admits mining is unbalanced.

>vocal minority insists image

>power-creep impossible to tamp despite being swiftly whacked elsewhere because "muh monster-hunter" tangential to the job

**Comment by Bawhoppen** [comment]
Remove GPS pls. Also remove the fact it's a miniature station rather than just a shitty bunker.

**Comment by AlexTheSysop42** [comment]
Elite hardsuit has better armor and shield suit has a shield.

Also the ammo (and beans) stockpile makes sense because it's a bunker, the idea is to bring enough supplies that you don't have to leave for years.

**Comment by WJohn** [comment]
I can re-add the extra food supplies, I suppose. But the armor really is incredibly strong, boasting 40/50/30/15 (melee, laser, bullet, energy) protection. This is roughly tied in durability with the HoS's personal hardsuit! And it's just sitting out in the open easy to steal with nothing protecting it. I think above all else that is the thing that must go.

**Comment by ExcessiveUseOfCobblestone** [comment]
d-d-d-d-dont you have to go into space to get this space suit though?

**Comment by Grazyn** [comment]
The amount of beans is excessive, it makes sense to tone it down. There are enough beans in there to make the kitchen irrelevant for an entire marathon round.

**Comment by Suicidalpickles** [comment]
although the syndie hardsuit does have some decent armor, can be bagged, and has a built-in jetpack, it has the risk of being lynched for even using it among the crew tbh. Don't see how its too problematic, considering its just a single suit. If the concern is over people panicking and assuming there's ops over seeing it about, maybe just give the suit a new color scheme? Bluesuit sounds pretty rad.

**Comment by blobbernaut** [comment]
Johnston, who the fuck cares if it's on par with the HOS hardsuit? Yeah, wow, some spaceloot's on par with a head's starting gear! SHUT DOWN EVERYTHING!

It's not even that good, anyway. You're a bit more durable to lasers and spears, you still go down to a taser or a few disablers. It's not good because of the armor, it's good because of the utility. Fits in a backpack and has a jetpack. 

It's kind of crazy you're allowed to change things when you do not even understand what is good about an item, holy shit.

**Comment by ExcessiveUseOfCobblestone** [comment]
You can disagree and not formulate it in such a rude manner.

**Comment by Repukan** [comment]
Yo this nigga eating beans in the abandoned bunker

**Comment by Anonmare** [comment]
If it's purely utility, then why not replace it with the soft syndicate spacesuit?

**Review by ShizCalev — DISMISSED** [review]

File: _maps/RandomRuins/SpaceRuins/deepstorage.dmm [hunk 1]
```diff
@@ -704,12 +707,31 @@
...
+	},
+/obj/item/reagent_containers/food/snacks/beans{
+	pixel_x = 1;
+	pixel_y = -3
+	},
+/obj/item/reagent_containers/food/snacks/beans{
+	pixel_x = 8;
+	pixel_y = -3
+	},
+/obj/structure/closet/crate{
```
Comment (ninjanomnom) [inline _maps/RandomRuins/SpaceRuins/deepstorage.dmm]: There seems to be 2 of these on the tile

File: _maps/RandomRuins/SpaceRuins/deepstorage.dmm [hunk 2]
```diff
@@ -720,12 +742,31 @@
...
```
Comment (ninjanomnom) [inline _maps/RandomRuins/SpaceRuins/deepstorage.dmm]: Same as above

**Review by ninjanomnom — CHANGES_REQUESTED** [review]

**Review by ninjanomnom — APPROVED** [review]

**Comment by blobbernaut** [comment]
Remember, you should have no incentive to visit space in a space station
game! Remove all z-levels!

**Comment by OmegaShoots** [comment]
>Let's remove any reason for anyone to want to explore off station, and instead just force them to stay on station looking for valids because everything ELSE on station that's fun also gets removed

:clap:

**Comment by blobbernaut** [comment]
basically

**Comment by epochayur** [comment]
\> +1, -18
\>merged anyway
🤔

**Comment by Barhandar** [comment]
Don't call straight nerf "rebalancing".

**Pull request merged**